release-notes/7.0/preview/api-diff/preview1/Microsoft.WindowsDesktop.App/7.0-preview1_System.Drawing.Design.md

```diff
@@ -0,0 +1,11 @@
+# System.Drawing.Design
+
+``` diff
+ namespace System.Drawing.Design {
+     public class UITypeEditor {
+-        public void PaintValue(object value, Graphics canvas, Rectangle rectangle);
++        public void PaintValue(object? value, Graphics canvas, Rectangle rectangle);
```

Comment:
I'm not sure if we should be listing all API that got NRTA'ed... NRTA do not change the API surface, rather make the contract more apparent. It probably makes sense where the change is in the other direction (i.e., nullable -> not nullable), as this is a breaking change.

dotnet/winforms is being annotated on ad-hoc basis as this activity is not a top priority for the team and is primarily driven by the community.